components/match2/wikis/rainbowsix/get_match_group_copy_paste_wiki.lua

```diff
@@ -16,7 +16,7 @@ local BaseCopyPaste = Lua.import('Module:GetMatchGroupCopyPaste/wiki/Base')
 ---@class RainbowsixMatch2CopyPaste: Match2CopyPasteBase
 local WikiCopyPaste = Class.new(BaseCopyPaste)
 
-local INDENT = WikiCopyPaste.Indent
+local INDENT = '\t'
```

Comment:
rath will not like that

Reply:
Good news, i dont like that either

Reply:
Good news, i dont like that either! idk why the dev module was doing that, should be fixed now